docs/vercel/package.json

```diff
@@ -4,14 +4,14 @@
   "version": "0.0.0",
   "type": "module",
   "scripts": {
-    "dev": "vite",
+    "start": "vite",
     "build": "vite build",
     "format": "prettier -w .",
     "preview": "vite preview",
-    "check": "check:svelte && check:prettier && check:eslint",
+    "check": "check:svelte && check:prettier && check:lint",
     "check:svelte": "svelte-check",
     "check:prettier": "prettier --check **/*.{ts,js,svelte,css,html,json}",
-    "check:lint": "eslint 'src/**/*.{ts,js,svelte}'"
+    "check:lint": "eslint src/**/*.{ts,js,svelte}"
```

Comment:
I forget exactly why this nuance is important, but quoting the glob ensures that `eslint` receives it as a string instead of the glob getting expanded to several arguments IIRC

Reply:
With the quote, i was getting the following error: ""No files matching the pattern "'src/**/*.{ts,js,svelte}'" were found.""